app/views/job_invocations/_user_input.html.erb

```diff
@@ -0,0 +1,21 @@
+<div id='preview_<%= target.id %>' class='collapse out'>
+  <%= preview_box(template_invocation, target) %>
+  <% if template_invocation.input_values.present? %>
+    <table class='datatable table table-striped table-bordered dataTable no-footer'>
```

Comment:
we just finished converting to table_css_classes helper provided by core (see https://github.com/theforeman/foreman_remote_execution/pull/313/files ) could you please update it here too?